src/v/cluster/health_monitor_frontend.h

```diff
@@ -58,6 +58,19 @@ class health_monitor_frontend {
     get_node_drain_status(
       model::node_id node_id, model::timeout_clock::time_point deadline);
 
+    /**
+     *  Return cluster health overview
+     *
+     *  Health overview is based on the information available in health monitor.
+     *  Cluster is considered as healthy when follwing conditions are met:
+     *
+     * - all nodes that are are responding
+     * - all partitions have leaders
+     * - cluster controller is present (_raft0 leader)
+     */
```

Comment:
Also this comment doesn't mention that cluster is not healthy if we couldn't refresh health information (does that mean that some nodes are not responding?)

Reply:
This is implicitly covered in `all cluster nodes are responding`, I've changed the comment to fix error in the fist item